typescript/cli/src/commands/warp.ts

```diff
@@ -402,9 +403,21 @@ export const check: CommandModuleWithContext<SelectWarpRouteBuilder> = {
       deployedRoutersAddresses,
     });
 
+    // Create altVmProviders dynamically for this call
```

Comment:
This is temporary given that expandWarpDeployConfig still expects altVmProviders